services/pane.js

```diff
@@ -91,36 +90,40 @@ function createPublishMessages(res) {
  * @returns {Element}
  */
 function createPublishActions(res) {
-  var today = moment().format('YYYY-MM-DD'),
+  var actions = dom.find('.publish-actions'),
```

Comment:
#1 – Could be a const for sure.
#2 – They all live there inside of `kiln-pane-template-elements` and could be shared, but each is directly selected.